### ISSUE TITLE
[Notification] Focus outline hard to see on low contrast notifications

### ISSUE THREAD

**Issue by elizabethsjudd** [carbon-design-system/carbon#5737]
## Environment

> Operating system

Mac OS X

> Browser

All

> Automated testing tool and ruleset

AVT 2 and https://www.ibm.com/able/guidelines/ci162/focus_visible.html

> Assistive technology used to verify

Keyboard

## Detailed description

While visible the focus indicator on low contrast notifications is visible it's very hard to see in the white theme.

> What version of the Carbon Design System are you using?

10.10.1

> What did you expect to happen?

The focus indicator to be the darker blue color

> What happened instead?

The focus indicator is white

> What WCAG 2.1 checkpoint does the issue violate?

https://www.ibm.com/able/guidelines/ci162/focus_visible.html

## Steps to reproduce the issue

1. Navigate to https://react.carbondesignsystem.com/?path=/story/notifications--inline
2. Check the `low contrast` knob
3. focus on the close button

> Please create a reduced test case in CodeSandbox

It's visible on the Carbon storybook site

## Additional information

<img width="1337" alt="Screen Shot 2020-03-27 at 4 18 55 PM" src="https://user-images.githubusercontent.com/1275779/77797435-6e13c600-7047-11ea-9dbd-9fc749ba9340.png">

**Comment by asudoh** [comment]
@elizabethsjudd Thank you for reporting - IIRC #2814 the low contrast version is a "legacy version" that exists primary for (use-case-wise) backward-compatibility. Need to see what the guidance would be in terms of a11y high contrast support and regular/low-contrast notification variants. CC @carbon-design-system/design

**Comment by laurenmrice** [comment]
Could we apply our regular `focus` token when the low contrast is applied?  That should solve this issue.

**Comment by elizabethsjudd** [comment]
@laurenmrice Is the low-contrast being depreciated? We have a breaking change coming up in the next week or so and if that's the case, I'd rather remove the low contrast on our end all together. My understanding was that both were going to remain supported in Carbon.

**Comment by laurenmrice** [comment]
No low contrast is not being deprecated, we give our users an option to use high contrast or low contrast notifications.
